department-of-veterans-affairs/abd-vro
# MCP-2231 Unit test python consumer

## What was the problem?

There were no tests for python RabbitMQ consumer.
Associated tickets or Slack threads:

- [MCP-2231](https://amida.atlassian.net/browse/MCP-2231)

## How does this fix it?[^1]

Testing to ensure consumer class is constructed properly and sets up queues. Tests that consumer modules are in place. 

## How to test this PR
- Step 1 Run python tests
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
